testsuite/features/support/database_handler.rb

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2024 SUSE LLC.
+# Licensed under the terms of the MIT license.
+require 'redis'
+
```

Comment:
If we are not sure if `DatabaseHandler` and `PrometheusHandler` will be used anywhere except `QualityIntelligence` at the moment. I would put these classes to `quality_intelligence.rb`. That's optional, I don't think we have a standards one class per file, I am just not a fan having this many files for one feature.

Reply:
DatabaseHandler it was used previously for code coverage, and I prefer to keep those handlers independent from QualityIntelligence, in case we want to re-use them for something else.

Reply:
ok